File: src/App/DTO/GetCategoriesRequest.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 * Параметры запроса категорий.
 */
class GetCategoriesRequest
{
    /**
     * @DTA\Data(field="language", nullable=true)
     * @DTA\Strategy(name="Object", options={"type":\App\DTO\LanguageType::class})
     * @DTA\Validator(name="TypeCompliant", options={"type":\App\DTO\LanguageType::class})
     * @var \App\DTO\LanguageType|null
     */
    public $language;

}

File: src/main/kotlin/ympa_kotlin_wiremock_server/models/GetCategoriesRequest.kt
@file:Suppress(
    "RemoveRedundantQualifierName",
    "unused",
)

package ympa_kotlin_wiremock_server.models

import com.fasterxml.jackson.annotation.JsonProperty

data class GetCategoriesRequest(
    @field:JsonProperty("language")
    val language: LanguageType? = null,

)
